build.sbt

```diff
@@ -20,9 +20,6 @@ scalacOptions := Seq(
   "-encoding", "UTF-8",
   "-feature",
   "-unchecked",
-  "-deprecation",
```

Comment:
please put back `-deprecation`, removal of `-Xfatal-warnings` should be enough

Reply:
Great